core/kazoo_services/src/kz_services_quantities.erl

```diff
@@ -543,8 +544,9 @@ calculate_qubicle_recipient_updates(JObj, Updates) ->
     of
         'false' -> Updates;
         'true' ->
-            Key = [<<"qubicle">>, <<"recipients">>],
-            [{Key, 1} | Updates]
+            BKey = [<<"qubicle">>, <<"recipients">>],
+            PKey = [<<"qubicle">>, <<"pro_recipients">>],
+            [{BKey, 1}, {PKey, 1} | Updates]
```

Comment:
@mark2600 same as above, could be simplified to:

Key = [<<"qubicle">>, <<(kz_json:get_ne_value(<<"pvt_qubicle_offering">>, JObj, <<"basic">>))/binary, "_recipients">>],
[{Key, 1}, | Updates]

Reply:
@mark2600 this is going to have the same problem I identified on the view, the JObj is the user document so it wouldn't have pvt_qubicle_offering.  BUT if you are inline with the qubicle payload on the user document I mentioned above then this just becomes the same as the view iteration over those keys.